diff --git a/apple/scripts/build-xcframework.sh b/apple/scripts/build-xcframework.sh
index 56bcb4ced0c25da2aaa170e3d2657ae2e4798ca7..3b8e417cc88384bf80aa93066aafa0c5c38a7094 100755
--- a/apple/scripts/build-xcframework.sh
+++ b/apple/scripts/build-xcframework.sh
@@ -217,7 +217,6 @@ build_slice() {
     -destination "generic/platform=${destination}" \
     -derivedDataPath "$DERIVED_DATA_PATH" \
     -configuration "$CONFIGURATION" \
-    -quiet \
     -disableAutomaticPackageResolution \
     -skipPackagePluginValidation \
     -skipMacroValidation \
